ReactiveCocoaTests/Swift/SignalProducerSpec.swift

```diff
@@ -218,17 +218,86 @@ class SignalProducerSpec: QuickSpec {
 			}
 		}
 
+		describe("ignoreValues") {
+			it("should not forward values from original producer") {
+				let producer: SignalProducer<Int, NoError> = SignalProducer<Int, NoError>(values: [ 1, 2 ]) |> ignoreValues
```

Comment:
That would probably do the trick for these lines in the test, but would break `then()`, which has the takes a `SignalProducer<T, E>` and returns a `SignalProducer<U, E>`.